Gems/PhysX/Code/Editor/Source/Components/EditorSystemComponent.cpp

```diff
@@ -236,9 +244,9 @@ namespace PhysX
             if (!resultAssetId.IsValid())
             {
                 // No file for the default material library, create it
-                const char* assetRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@projectsourceassets@");
+                const char* projectRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@projectroot@");
                 AZStd::string fullPath;
-                AzFramework::StringFunc::Path::ConstructFull(assetRoot, DefaultAssetFilePath, assetExtension.c_str(), fullPath);
+                AzFramework::StringFunc::Path::ConstructFull(projectRoot, DefaultAssetFilePath, assetExtension.c_str(), fullPath);
```

Comment:
I have fixed this up in this PR https://github.com/o3de/o3de/pull/4186/files#diff-76b974e95411b22324f738157a9b1e6f1cf0d24bbaa28d90925bf129ec0d3aaa

Reply:
Cool, please revert it in that PR, I'll update it here.